source/electrical power systems.md

```diff
@@ -0,0 +1,41 @@
+# Electrical Power Systems
+
+Electrical trains need to be powered.
+In almost all cases, the best way to do this is not by having trains carry their own energy supply, as diesel and coal trains do, but to be connected to the electrical grid to receive externally-generated power continuously.
+Such power systems can be broken down into overhead and "third rail" systems.
+
+## Overhead Catenary Wire
+
+Catenary wire is strung up over the tracks.
+
+Rolling stock which needs power is connected to the overhead wire with a *pantograph*.
+
+Modern overhead wire wire systems are typically at higher voltages AC, such as 25 kHz and 50 kHz.
```

Comment:
```suggestion
Modern overhead wire systems are typically at higher voltages AC, such as 25 kHz and 50 kHz.
```

Reply:
Also, subways and light rail with catenary often still use DC.  AC is only used for faster metros (e.x. 80-100+ mph) generally.

Also, kHz is a frequency, not voltage.  I think you mean 25 kV and 50 Hz.

Reply:
> Also, kHz is a frequency, not voltage. I think you mean 25 kV and 50 Hz.

Oh whoops, yes 🤦. I was thinking of the the "what voltage to use for trans cons" things, and I guess 50 kV is not actually common at all.

Reply:
```suggestion
Modern overhead wire systems for mainline rail are typically at higher voltages AC, such as 25 kV.
```